spring-cloud-dataflow-server-cloudfoundry-docs/src/main/asciidoc/cf-tasks.adoc

```diff
@@ -2,112 +2,125 @@
 = Tasks on Cloud Foundry
 
 Spring Cloud Data Flow's task functionality exposes new task capabilities within
-the Pivotal Cloud Foundry runtime. It is important to note that the current underlying PCF
+the Pivotal Cloud Foundry runtime.
+
+IMPORTANT: The current underlying PCF
 task capabilities are considered experimental for PCF version versions less than 1.9.  See
```

Comment:
We could remove all the references to PCF 1.9 and the experimental call-outs. It is a very old release and it is not a supported version at Pivotal. Not anymore.